code/datums/status_effects/debuffs.dm

```diff
@@ -968,3 +968,25 @@
 /datum/status_effect/cloudstruck/Destroy()
 	. = ..()
 	QDEL_NULL(mob_overlay)
+
+
+//~~~LC13 General Debuffs~~~
+
+/datum/status_effect/rendRedArmor
```

Comment:
Datum names(just like variables) should be in lower_case.
CamelCase is reserved for procs.